docs/workflow_solutions/getting_containers.md

```diff
@@ -287,6 +287,66 @@ python python_test.py
 
 More lessons on Docker can be found in this link: [Introduction to Docker](https://christinalk.github.io/docker-introduction/) and [Docker Documentation](https://docs.docker.com/reference/dockerfile/).
 
+## How to Access and Use Community Containers in the GitLab Container Registry
+
+In the UAB Research Computing [GitLab instance](../account_management/gitlab_account.md#uab-gitlab-registration), we provide certain community containers—prebuilt, these containers are standardized environments that grant you access to a variety of software tools. in the sections below, we share information for what you will find in the gitlab community container repo, and how you can access them.
+
+### Software We Support in Building Community Containers
+
+We provide community containers that cover a broad range of software to support your research needs. In general, the containers we have in our container registry are for software and applications with;
+
+1. Specific licensing restrictions to ensure compliance with licensing terms.
+1. Compatibility/dependency issues: For example, when software require package or kernel versions that are incompatible with Cheaha, or have other system-specific dependencies.
+1. Software not available on Docker Hub: When you need to containerize software and its dependencies for sharing within your group or organization, and you are unable to get one anywhere open-source containers can be found (DockerHub, etc).
+
+### Where Can I Find UAB Research Computing Community Containers
+
+Our community containers are housed in the university’s GitLab repository. You can access them at <https://gitlab.rc.uab.edu/rc-data-science/community-containers>
```

Comment:
"university's" should be "ours". Technically you are correct, but we can safely be more specific here.